skimage/future/graph/rag.py

```diff
@@ -461,20 +456,30 @@ def draw_rag(labels, rag, img, border_color=None, node_color='#ffff00',
     >>> img = data.coffee()
     >>> labels = segmentation.slic(img)
     >>> g =  graph.rag_mean_color(img, labels)
-    >>> out = graph.draw_rag(labels, g, img)
+    >>> lc = graph.show_rag(labels, g, img)
+    >>> cbar = plt.colorbar(lc)
     """
+
     if not in_place:
         rag = rag.copy()
 
-    if desaturate:
-        img = color.rgb2gray(img)
-        img = color.gray2rgb(img)
-
+    if ax is None:
+        fig, ax = plt.subplots()
     out = util.img_as_float(img, force_copy=True)
-    cc = colors.ColorConverter()
 
-    edge_color = cc.to_rgb(edge_color)
-    node_color = cc.to_rgb(node_color)
+    if img_cmap is None:
```

Comment:
I believe @soupault's version to be both the fastest, most concise, and most standard (ie that pattern exists elsewhere in skimage)